migrate/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Migrate your data to Timescale
+excerpt: Migrate existing database to Timescale
+products: [cloud]
+keywords: [data migration, postgresql, RDS]
+tags: [ingest, migrate, RDS]
+---
+
+# Migrate your data to Timescale
+
+There are a number of different ways to migrate your data to Timescale. Which
+option you choose depends on a few different factors, the most important of
+which are:
+
+- How much downtime you can afford (minutes, or hours?)
+- How much data you are migrating (megabytes, or terabytes?)
+- Where you are migrating your data from (PostgreSQL, TimescaleDB, Influx, or MySQL?)
+
+If you are using PostgreSQL or TimescaleDB and can afford to take your
+application offline for a few hours, the simplest option is to migrate data
+from another database into Timescale using PostgreSQL's `pg_dump` and
+`pg_restore` commands.
+You can also use these tools to migrate your data from Managed Service for
+TimescaleDB, from a self-hosted TimescaleDB instance, or from another
+PostgreSQL database, including Amazon RDS. Consult the guide on [migrating with
+pg_dump and pg_restore][pg-dump-restore].
+
+If you are looking for a low-downtime alternative (downtime on the order of
+minutes), are willing to modify your ingestion code, and the bulk of your data
+is stored in time-series tables, you can use the [dual-write and backfill][dual-write]
+strategy for a low-downtime migration. This strategy also works if your source
```

Comment:
I think your suggestion is good in general. I'm not going to solve it exactly like how you suggested though. Instead I'm going to re-write this page so that it doesn't mention `source` anymore, and put the information about "what are `source` and `target` into the step-by-step instruction pages.